apis/v1alpha2/gateway_types.go

```diff
@@ -196,9 +196,48 @@ type GatewayConditionType = v1beta1.GatewayConditionType
 type GatewayConditionReason = v1beta1.GatewayConditionReason
 
 const (
-	// This condition is true when the controller managing the
-	// Gateway has scheduled the Gateway to the underlying network
-	// infrastructure.
+	// This condition indicates whether a Gateway have generated some
```

Comment:
Is there a reason why we don't alias these to the ones defined in `v1beta1` like we did with the types? Should help reducing duplication, wdyt?

Reply:
Yes, in general unless there's a reason a split needs to exist, add to beta and use in alpha.

Reply:
Cool, I haven't dug deeper to check if all of them are exactly the same, but I think there are some. I can open an issue after I have a closer look and send in a follow-up. Thanks!